src/NServiceBus.AcceptanceTests/Sagas/When_forgetting_to_set_a_corr_property.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace NServiceBus.AcceptanceTests.Sagas
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+
+    //note: this test will be obsolete if/when we implement https://github.com/Particular/NServiceBus/issues/313
+    public class When_forgetting_to_set_a_corr_property : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_blow_up()
+        {
+            await Scenario.Define<Context>()
+                .WithEndpoint<NullPropertyEndpoint>(b => b.When(bus => bus.SendLocalAsync(new StartSagaMessage
+                {
+                    SomeId = Guid.NewGuid()
+                })))
+                .AllowExceptions(ex => ex.Message.Contains("All correlated properties must have a non null or empty value assigned to them when a new saga instance is created"))
+                .Done(c => c.Exceptions.Any())
+                .Run();
+        }
```

Comment:
So any old exception will pass this test? The only way it can fail is to time out?